README.md

```diff
@@ -203,6 +203,12 @@ cmake ..
 make
 ```
 
+Custom library locations can be provided with the following cmake variables:
++ `OPENNMT_TOKENIZER_ROOT` for **OpenNMT_Tokenize** path.
++ `CMAKE_PREFIX_PATH` for **ICU** path.
```

Comment:
For consistency with the other libraries, maybe we could use `ICU_ROOT` here. It is supported by `FindICU.cmake`.

Reply:
It does not seems to work, I have a ICU in system path and another user provided one and the system ICU gets chosen when using `ICU_ROOT`.